src/components/Hive.vue

```diff
@@ -16,10 +25,32 @@ const otherLetters = ref(
 );
 let userGuess = ref("");
 
+const onKeyPress = (e: KeyboardEvent) => {
+  if (e.key == "Enter") {
```

Comment:
I guess these comments only submitted now since I added a new comment through the `request changes` dialog :man_facepalming: 
For the record I'm pretty sure I added the first comment the day you opened the pr, and the second one a few days later. What a pain

Reply:
It's been a fairly long time now so I've gone ahead and done this myself. Thanks for your contribution though!